products/billingbudget/terraform.yaml

```diff
@@ -14,52 +14,64 @@
 --- !ruby/object:Provider::Terraform::Config
 overrides: !ruby/object:Overrides::ResourceOverrides
   Budget: !ruby/object:Overrides::Terraform::ResourceOverride
+    docs: !ruby/object:Provider::Terraform::Docs
+      warning: |
+        If you are using User ADCs (Application Default Credentials) with this resource,
+        you must specify a `billing_project` and set `user_project_override` to true
+        in the provider configuration. Otherwise the Billing Budgets API will return a 403 error.
+        Your account must have the `serviceusage.services.use` permission on the
+        `billing_project` you defined.
     id_format: '{{name}}'
-    import_format: ['{{name}}']
+    exclude_import: true
     examples:
       - !ruby/object:Provider::Terraform::Examples
         name: 'billing_budget_basic'
-        min_version: beta
         primary_resource_id: 'budget'
         vars:
           display_name: 'Example Billing Budget'
         test_env_vars:
           billing_acct: :BILLING_ACCT
+        skip_test: true
```

Comment:
Why are all these tests skipped and converted to being handwritten? Nothing seems to have changed significantly in the handwritten ones.

Reply:
Import tests were failing.

Reply:
Oh, huh, I guess we never actually added code to skip the import step if the resource doesn't support import. Huh! Mind commenting something to that effect?

Reply:
I introduced a new field that excludes import step test from generated tests.